### PR TITLE
Update TooltipTrigger docs to mention trigger and tooltip ordering (#1245)

### LINKED ISSUE #1245: Update TooltipTrigger docs to mention trigger and tooltip ordering
# 🙋 Documentation Request

From https://github.com/adobe/react-spectrum/issues/1212#issuecomment-722853352. We should mention that the Tooltip and trigger elements need to be provided to the TooltipTrigger in a specific order like we do for DialogTrigger (https://react-spectrum.adobe.com/react-spectrum/DialogTrigger.html#content) and MenuTrigger (https://react-spectrum.adobe.com/react-spectrum/MenuTrigger.html#content)

## 🧢 Your Company/Team

RSP

### PR DESCRIPTION
**Pull request by reidbarber**
Closes #1245 

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [x] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [x] Filled out test instructions.
- [x] Updated documentation (if it already exists for this component).
- [x] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 📝 Test Instructions:

Check docs at React Spectrum -> Tooltip -> Content

### CODE REVIEW

**Review by LFDanLu — DISMISSED** [review]
LGTM

File: packages/@react-spectrum/tooltip/docs/Tooltip.mdx [hunk 1]
```diff
@@ -53,7 +53,7 @@ keywords: [tooltip]
 
 ## Content
 
-Tooltips accept content as children. All content should be internationalized.
+The TooltipTrigger accepts exactly two children: the Tooltip and the element which triggers the display of the Tooltip. The trigger element must be the first child passed into the TooltipTrigger. All content should be internationalized.
```
Comment (snowystinger) [inline packages/@react-spectrum/tooltip/docs/Tooltip.mdx:56]:
```suggestion
The TooltipTrigger accepts exactly two children: the element which triggers the display of the Tooltip and the Tooltip itself. The trigger element must be the first child passed into the TooltipTrigger. All content should be internationalized.
```

Just want to be consistent in the order that we talk about them so it naturally leads the user to put them in the right order. You'll notice the DialogTrigger sentence does the same

Comment (reidbarber) [reply packages/@react-spectrum/tooltip/docs/Tooltip.mdx:56]: Good catch. I'll correct this in MenuTrigger too.

**Review by snowystinger — APPROVED** [review]
Great! Thanks for catching that in menu trigger as well

**Review by dannify — APPROVED** [review]

**Pull request merged**